tools/c7n_azure/c7n_azure/resources/sqldatabase.py

```diff
@@ -30,3 +39,176 @@ class resource_type(ArmResourceManager.resource_type):
             manager_name='sqlserver',
             annotate_parent=True
         )
+
+
+class BackupRetentionPolicyFilter(Filter):
+
+    # TODO: consolidate with MetricFilter.op?
+    number_ops = {
+        'eq': operator.eq,
+        'equal': operator.eq,
+        'ne': operator.ne,
+        'not-equal': operator.ne,
+        'gt': operator.gt,
+        'greater-than': operator.gt,
+        'ge': operator.ge,
+        'gte': operator.ge,
+        'le': operator.le,
+        'lte': operator.le,
+        'lt': operator.lt,
+        'less-than': operator.lt
+    }
+
+    def __init__(self, operations_property, data, manager=None):
+        super(BackupRetentionPolicyFilter, self).__init__(data, manager)
+        self.operations_property = operations_property
+
+    def list_backup_retention_policies(self, i):
+        client = self.manager.get_client()
+        list_operation = getattr(client, self.operations_property).list_by_database
+
+        resource_group_name = i.get('resourceGroup')
+        server_name = BackupRetentionPolicyFilter.get_sql_server_name(
+            i[ChildResourceQuery.parent_key])
+        database_name = i.get('name')
+
+        list_response = list_operation(
+            resource_group_name, server_name, database_name)
+        return list_response
+
+    def __call__(self, i):
+        policies = self.list_backup_retention_policies(i)
+        return self.filter_with_policies(i, policies)
+
+    def filter_with_policies(self, i, policies):
```

Comment:
nit: can we use retention_policies? :) Policies is an overloaded term in Custodian